src/static/css/page.css

```
@@ -643,7 +643,7 @@ figure .fig-desktop {
   left: -1rem;
 }
 
-.code-block pre {
+.code-block, pre {
```

One pattern I usually do is to separate selectors by a newline. We could enforce this in the linter if needed.
```suggestion
.code-block,
pre {
```

---

Always thought this was overkill. This bug has convinced me otherwise!

Raised #1053

---

Especially for selectors that are very similar, it's helpful to scan through and see the changes line up vertically, for example:

```css
h1 + .subtitle,
h2 + .subtitle,
h3 + .subtitle {
```